keyboards/handwired/macroboard/rules.mk

```diff
@@ -0,0 +1,16 @@
+# MCU name
+MCU = STM32F401
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
```

Comment:
Totally optional, but maybe....
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
```

Reply:
How can i test the bootmagic (whatever it is)?

Reply:
Non the less i did requested changes

Reply:
hold top left key when plugging in usb, it should then go into the bootloader and allow flashing

Reply:
Works :)